### PR TITLE
Added CHANGELOG.md

### LINKED ISSUE #415: Add release notes / changelog

### PR DESCRIPTION
**Pull request by rjmcneill**
Closes #415 
Up to date as of the conclusion of the last sprint (Version 0.7.0)

### CODE REVIEW

**Comment by djmiley** [comment]
Great work @rjmcneill !

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+- Issue where switching from live data to generated would cause a basket of live data to be added to the generated data
+- Latest price annotation mismatch
+- Issue where area series y0 value appeared above the x-axis
+
+## [0.2.0] - 12-08-2015
+### Added
+- Close price annotation callout label on y-axis
+- Live Coinbase data source 
+- Historical Coinbase data source
+### Changed
```
Comment (jleft) [inline CHANGELOG.md]: Need a new line before this, otherwise it won't format properly

**Comment by jleft** [comment]
Nice, thanks @rjmcneill. Happy to merge once the comment is addressed and the branch is rebased :+1: 

Just curious - how did you create the list? Using git log? I'm wondering if it's worth automating this in some way; at some point we may want to add this to the [GitHub releases](https://help.github.com/articles/about-releases/) too.

**Comment by rjmcneill** [comment]
@jleft I used the milestones and the issues within them.
I read up on how to do it, and it basically said not to dump the commit messages into a changelog.
From there, it was just a matter of judging what was an added feature, change, or fix.

If we can't automate it, I'm happy to keep it up to date and the end of each sprint.

**Comment by jleft** [comment]
Cool, manually updating sounds good to me.

We could probably do something like this:

```bash
git log 0.6.0..0.7.0 --merges --format="%b" | awk '{print "- " $0}'
```

to view all the merges between two tags - could be a good starting point - assuming that we stick to one issue per PR, then manually update/categorise as required.

**Comment by rjmcneill** [comment]
Yea, that would be a nice solution :)
Well In that case, I'm happy to do that after each sprint

**Pull request merged**